tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -220,20 +221,14 @@ stages:
           AnalyzeTargetGlob: '+:file|$(Build.ArtifactStagingDirectory)\**\*.dll;-:file|$(Build.ArtifactStagingDirectory)\**\DirectML.dll'
         continueOnError: true
 
-      - task: DeleteFiles@1
```

Comment:
why delete this task

Reply:
Because I need the dlls to test cloud build in following step.
Why we are dropping those files for python 3.7 exclusively?